holidays/countries/mauritius.py

```
@@ -0,0 +1,197 @@
+    HolidayBase,
+    ChineseCalendarHolidays,
+    ChristianHolidays,
+    HinduCalendarHolidays,
+    InternationalHolidays,
+    IslamicHolidays,
+    StaticHolidays,
+):
+    """Mauritius holidays.
+
+    References:
+        * [Public Holidays Act Consolidated 1991](https://web.archive.org/web/20250618211720/https://natlex.ilo.org/dyn/natlex2/natlex2/files/download/101480/MUS101480.pdf)
```

Pylint (3.3.7)

[convention] Line too long (172/100)

(C0301)